v3/plugins/ms-vscode/node-debug2/1.33.0a/meta.yaml

```diff
@@ -17,5 +17,5 @@ spec:
     name: vscode-node-debug
     memoryLimit: '512Mi'
   extensions:
-  - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-node-debug/node-debug-1.44.5.vsix
+  - https://download.jboss.org/jbosstools/vscode/3rdparty/vscode-node-debug/node-debug-1.41.1.vsix
```

Comment:
I thought we would do 1.33.0b and automigrate from the previous version to do "bugfix" versions?

Reply:
Also, shouldn't we deprecate some old versions when we introduce a new one?

Reply:
As for me there is no reason to create 1.33.0b as 1.33.0a doesn't work at all. Do you see some benefits to have old 1.33.0a plugin?
As i know we mark some version as deprecate when we have more than 3 version, and node-debug2 plugin has 3 versions.

Reply:
I think it's incorrect to have the 1.33.0a plugin have no extensions from that actual version.In this case I say let's deprecate 1.33.0 entirely (a and b) and have the `autoMigrate` set to `latest`.

Reply:
Marked 1.33.0a as deprecated and set `automigrate` to latest